src/server/internalServer.cpp

```diff
@@ -630,7 +630,9 @@ std::unique_ptr<Response> InternalServer::handle_search(const RequestContext& re
     renderer.setProtocolPrefix(m_root + "/");
     renderer.setSearchProtocolPrefix(m_root + "/search?");
     renderer.setPageLength(pageLength);
-    auto response = ContentResponse::build(*this, renderer.getHtml(), "text/html; charset=utf-8");
+    const bool rawMode = rawModeWasRequested(request);
```

Comment:
In #731 I introduce the `format=xml` for opensearch results. (without being sure that `format` is the right name).
I wonder if there is something to define in common or not.

Reply:
If we drop the old taskbar, I would prefer to undo the `?mode=raw` hacks for the `/random` and `/search` endpoints